bears/java/CheckstyleBear.py

```diff
@@ -11,7 +11,7 @@
     'android-check-hard': 'https://raw.githubusercontent.com/noveogroup/' +
     'android-check/master/android-check-plugin/src/main/resources/' +
     'checkstyle/checkstyle-hard.xml',
-    'geosoft': 'http://geosoft.no/development/geosoft_checks.xml'}
+}
```

Comment:
I dont see it covered by https://api.coala.io/en/latest/Developers/Codestyle.html#line-continuation , and imo that is not good practise.